src/utils.js

```diff
@@ -1,3 +1,4 @@
+import prettier from "prettier";
```

Comment:
Should I import it like this or use `requireModule` from `index.js`?

Reply:
I'm thinking let's use `requireModule` and then we can come back to this. It's related to the discussion in #133.